sharding-core/sharding-core-common/src/test/java/org/apache/shardingsphere/core/rule/ShardingRuleTest.java

```
@@ -328,6 +329,33 @@ public void assertGetShardingLogicTableNames() {
         ShardingRule actual = createMaximumShardingRule();
         assertThat(actual.getShardingLogicTableNames(Arrays.asList("LOGIC_TABLE", "BROADCAST_TABLE")), CoreMatchers.<Collection<String>>is(Collections.singletonList("LOGIC_TABLE")));
     }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void assertConstructShardingRuleWithNullShardingRuleConfiguration(){
+        try {
+        new ShardingRule(null, createDataSourceNames());
+            fail("should throw a IllegalArgumentException when execute ShardingRule's constructor with null ShardingRuleConfiguration");
+        } catch (IllegalArgumentException ignored){
+        } catch (Exception ex){
+            fail("should throw a IllegalArgumentException when execute ShardingRule's construction with null ShardingRuleConfiguration");
+        }
+    }
+
+    @Test(expected = IllegalArgumentException.class)
```

Ignore exception or @Test(expected= )

---

I'll do it later.